src/main/java/net/pms/database/MediaTableAudiotracks.java

```diff
@@ -132,6 +132,20 @@ private static void upgradeTable(Connection connection, Integer currentVersion)
 						LOGGER.trace("Indexing column RATING on table " + TABLE_NAME);
 					}
 					break;
+				case 7:
+					Connection con = DATABASE.getConnection();
```

Comment:
There is already the `connection` variable to use here, we don't have to create a new one

Reply:
I needed a connection without autocommit ...  OK, I could shuffle the autocommit flag. Not sure if this is more readable ...